### DISCUSSION TITLE
Add good data sources to docs guides

### DISCUSSION THREAD

**Discussion by mcrascal** [evidence-dev/evidence#2360] [General]
**Feature Description**
I'm trying out Evidence and I want to find some interesting data to work with, it'd be great to have some third party data sets listed for me in the docs.

**Comment by davidgasquez** [comment]
In case it helps, I maintain [a curated list of Open Datasets and Dataset Indexes](https://publish.obsidian.md/davidgasquez/Open+Data#Open+Datasets).

**Comment by mcrascal** [comment]
Epic, thank you! 

What an awesome obsidian site btw. Especially like your section on dashboards 👀
